### DISCUSSION TITLE
pytest用的python版本和我自己的python版本不太一样？

### DISCUSSION THREAD

**Discussion by lhyyy1** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#44] [Q&A] [answered]
如图所示，pytest显示python版本是2.7并且报语法错误，但实际版本是3.8
![GAA%{SF~SNE(({)BY59N_A3](https://user-images.githubusercontent.com/120730404/210033158-998b5cb3-f150-4fc8-a10d-5e81355ac17a.png)

**Comment by TANSixu** [comment, marked as answer]
Hi，请尝试运行
```
python3 -m pytest xxxxxxxx
```
这看起来应该是被默认引导到python2去了。

**Reply by lhyyy1** [reply]
现在可以了，谢谢！
